langchain-core/src/language_models/base.ts

```diff
@@ -216,7 +216,7 @@ export interface FunctionDefinition {
    * To describe a function that accepts no parameters, provide the value
    * `{"type": "object", "properties": {}}`.
    */
-  parameters: Record<string, unknown>;
+  parameters?: Record<string, unknown>;
```

Comment:
Likewise this change is because of: https://github.com/openai/openai-node/commit/c4fadb4513fc2e35cf0e4b9512ec39a6bd0e42dd#diff-30f3fcee562aad291d98f76d670f7cef5920b2ad946be42c3f6d02297767ec2cR772-R774